devices/tuya.js

```diff
@@ -4114,4 +4114,32 @@ module.exports = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_r32ctezx']),
+        model: 'TS0601_fan_switch',
+        vendor: 'TuYa',
+        description: 'Fan switch',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            tuya.exposes.switch(),
+            tuya.exposes.countdown().withValueMin(0).withValueMax(43200).withUnit('s').withDescription('Max ON time in seconds'),
+            exposes.numeric('fan_speed', ea.STATE_SET).withValueMin(1).withValueMax(5).withValueStep(1)
+                .withDescription('Speed off the fan'),
+            exposes.enum('power_on_behavior', ea.STATE_SET, ['off', 'on']).withDescription('Relay state'),
```

Comment:
Can you confirm this is the Relay state after power recovery?

Reply:
Yep, it is a state after power recovery